source/js/buyers-guide/template-js-handler/parallax-background.js

```diff
@@ -0,0 +1,39 @@
+import { gsap } from "gsap";
+import { ScrollTrigger } from "gsap/ScrollTrigger";
+
+export default () => {
+  gsap.registerPlugin(ScrollTrigger);
+  const parallaxBackground = document.querySelector("#pni-parallax-background");
+
+  if (parallaxBackground) {
+    const svgShapes = document.querySelectorAll("#pni-parallax-background svg");
+    svgShapes.forEach((shape) => {
+      gsap.to(shape, {
+        y: -1000,
+        ease: "none",
+        scrollTrigger: {
+          trigger: "body",
+          start: "top top", // when the top of the trigger hits the top of the viewport
+          end: "bottom bottom",
```

Comment:
Does this mean we don't use the animation for scrolling the bottom of the shapes wrapper to the top of the scroller/viewport? It seems like a change of the scroll behaviour while the element is in view, no?

Reply:
scrollTrigger start/bottom is kinda confusing to explain. If you want a visual understanding of what is happening I suggest adding `markers: true` to the object to see how the trigger and scrubbing works. The goal is the start of the scroll animation is the top of the body and bottom is the body of the animation and the animation progress is based on the scroll position. like https://loremipsum.io/